src/filtering/filtering.py

```diff
@@ -1,6 +1,6 @@
 import networkx as nx
 
-from src.filtering_handler import FilteringHandler
+from src.filtering.filtering_handler import FilteringHandler
```

Comment:
The word filtering can be misleading in a code for processing time series since there are "filters" in signal processing and stochastics for this object. I would rename everything to "select" (then instead of filtering, use selection).